src/magic_folder/test/test_cli.py

```diff
@@ -55,3 +67,20 @@ def test_good(self):
         for d in [d0, d1, d2]:
             self.assertTrue(d.called)
             self.assertEqual(d.result, "we listened")
+
+
+class TestInitialize(SyncTestCase):
+    """
+    Confirm operation of 'magic-folder initialize' command
+    """
+
+    def setUp(self):
+        super(TestInitialize, self).setUp()
+        self.temp = FilePath(self.mktemp())
+
+    def tearDown(self):
+        super(TestInitialize, self).tearDown()
+        rmtree(self.temp.path, ignore_errors=True)
```

Comment:
`FilePath.remove` is also a thing.  Additionally, `mktemp` creates the temporary directory beneath `_trial_temp` and trial deletes old `_trial_temp` directories when it starts up a new run.  This behavior makes me feel okay about tests leaving behind all of their temporary files.  They won't grow without bound and it's nice to automatically have them available for inspection instead of having to go out of your way to get them.